src/server/index.js

```diff
@@ -13,6 +13,7 @@ import { getCacheEngine } from '~/src/server/common/helpers/session-cache/cache-
 const isProduction = config.get('isProduction')
 
 async function createServer() {
+  const cacheEngine = getCacheEngine()
```

Comment:
wondering if it would be clearer if the type of cache engine (redis/catbox etc) was passed in as a param at this point. `getCacheEngine()` is a bit of a blackbox?
something like`getCacheEngine('redis')` or `getCacheEngine(config.get('cache.engine'))`

the logic for picking the engine etc goes into config.js